src/OnlineSales/Infrastructure/PluginManager.cs

```diff
@@ -1,15 +1,17 @@
 ﻿// <copyright file="PluginManager.cs" company="WavePoint Co. Ltd.">
 // Licensed under the MIT license. See LICENSE file in the samples root for full license information.
 // </copyright>
+using System.Diagnostics;
 using System.Reflection;
 using System.Runtime.Loader;
+using Microsoft.AspNetCore.Hosting.Server;
 using OnlineSales.Interfaces;
 
 namespace OnlineSales.Infrastructure;
 
 public static class PluginManager
 {
-    private static readonly string PluginsFolder = "plugins";
+    private static readonly string PluginsFolder = "./bin/debug/net7.0/plugins";
```

Comment:
@WASandaruwan, this is hardcode, you can not make it this way as it will not work in production environment. It will only work if you run it from Visual Studio. You should never hardcode it this way, instead you should detect the current folder of the dll which is being executed, i.e. OnlineSales.dll and company that path with "plugins" folder.

Reply:
@peterliapin  - This one resolved